packages/scandipwa/src/component/StoreInPickUpPopup/StoreInPickUpPopup.container.js

```diff
@@ -133,11 +140,22 @@ export class StoreInPickUpContainer extends PureComponent {
         this.setState({ stores: [] });
     }
 
+    closeOverlay() {
+        const {
+            hideActiveOverlay,
+            goToPreviousNavigationState,
+            navigationState: { onCloseClick } = {}
+        } = this.props;
+
+        onCloseClick();
```

Comment:
BTW, if `navigationState` can be empty (you assume it can be empty since you added the default value `{}`), then `onCloseClick` can be `undefined`.
Please add a check here.